Marlin/Marlin_main.cpp

```diff
@@ -1713,27 +1713,37 @@ static void clean_up_after_endstop_or_probe_move() {
       z_dest -= zprobe_zoffset;
 
     if (z_dest > current_position[Z_AXIS]) {
-      float old_feedrate = feedrate;
-      feedrate = homing_feedrate[Z_AXIS];
```

Comment:
Excep of line 1662, 1674, 1667 in `do_blocking_move_to()`. Please look it up. (https://github.com/MarlinFirmware/Marlin/blob/RCBugFix/Marlin/Marlin_main.cpp#L1638)